application/ui/components/api-resource.jsx

```diff
@@ -51,13 +51,27 @@ module.exports = React.createClass({
 
         if (resource) {
             window.document.title = resource.title;
-            resourceComponent = (
+
+            // Allow the docs to use either methods or endpoints, but not both.
+            if (typeof(resource.endpoints) === 'undefined')
```

Comment:
This supports the old way of doing the lively docs, using "methods" rather than "endpoints", but I'm open to suggestions about better ways to do it.